examples/riemann.config

```diff
@@ -10,8 +10,14 @@
   (ws-server {:host host :port 8087})
 )
 
-(mbeans/instrumentation {:interval 10 :mbeans [{:mbean "java.lang:type=Memory" :property :HeapMemoryUsage :attribute :used}
-                                               {:mbean "java.lang:type=Runtime" :property :Uptime}]})
+(mbeans/instrumentation
+  {:interval 10
+   :mbeans [{:mbean "java.lang:type=Memory" :property :HeapMemoryUsage :attribute :used}
+            {:mbean "java.lang:type=Runtime" :property :Uptime}
+            {:mbean "java.lang:type=GarbageCollector,name=ParNew"
+             :traverse '(:LastGcInfo :memoryUsageAfterGc (keyword "Code Cache") :value :used)}
```

Comment:
passing a quoted list doesn't work for me when using `keyword` in the traversal:

```
;:traverse '(:LastGcInfo :memoryUsageAfterGc (keyword "PS Eden Space") :value :used)
2016-03-24T09:25:51.000Z java.lang:type=GarbageCollector,name=PS Scavenge:LastGcInfo:memoryUsageAfterGc(keyword "PS Eden Space"):value:used N/A
```

Passing a vector instead works:

```
;:traverse [:LastGcInfo :memoryUsageAfterGc (keyword "PS Eden Space") :value :used]
2016-03-24T09:27:10.000Z java.lang:type=GarbageCollector,name=PS Scavenge:LastGcInfo:memoryUsageAfterGc:PS Eden Space:value:used 0
```